react/features/participants-pane/actions.native.js

```diff
@@ -81,3 +82,26 @@ export function setVolume(participantId: string, volume: number) {
         volume
     };
 }
+
+/**
+ * Displays the breakout room participant menu.
+ *
+ * @param {Object} room - The room the participant is in.
+ * @param {string} participantJid - The jid of the participant.
+ * @param {string} participantName - The display name of the participant.
+ * @returns {Function}
+ */
+export function showRoomParticipantMenu(room: Object, participantJid: string, participantName: string) {
+    return openDialog(RoomParticipantMenu, { room,
+        participantJid,
+        participantName });
+}
+
+/**
+ * Hides the breakout room participant menu.
+ *
+ * @returns {Function}
+ */
+export function hideRoomParticipantMenu() {
```

Comment:
No need for helpers, sheets can only be shown one at a time, so `hideSheet` takes no parameters.